pages/payment.js

```diff
@@ -0,0 +1,281 @@
+              <input
+                data-cy="full-name"
+                type="text"
+                id="name"
+                placeholder="Alenio Hasslacherio"
+                onChange={checksTypeText}
+              ></input>
+            </span>
+            <span>
+              <label htmlFor="email">Email</label>
+              <input
+                id="email"
+                type="email"
+                placeholder="alenio@teppich.com"
+              ></input>
+            </span>
+            <span>
+              <label for="address">Address</label>
+              <input
+                id="address"
+                type="text"
+                placeholder="Awesome Str. 1/2/3"
+              ></input>
+            </span>
+            <span>
+              <label for="zip">Zip-code</label>
+              <input
+                id="zip"
+                placeholder="2320"
+                type="text"
+                maxLength="4"
+                onChange={checksTypeNumber}
+              ></input>
+            </span>
+            <span>
+              <label for="city">City</label>
+              <input id="city" placeholder="Raccoon-city" type="text"></input>
+            </span>
+          </div>
+          <div className="payment">
+            <h3>Payment info</h3>
+            <div className="cards">
+              <p>Accepted cards</p>
+              <img src="/visa.png" alt="visa"></img>
+              <img src="/alenio.png" alt="jaegermeister"></img>
+            </div>
+            <span>
+              <label for="namecard">Name on card</label>
+              <input
+                id="namecard"
+                type="text"
+                placeholder="Alenio B. Hasslacherio"
+                onChange={checksTypeText}
+              ></input>
+            </span>
+            <span>
+              <label for="cardnumber">Credit card number</label>
+              <input
+                id="cardnumber"
+                type="text"
+                placeholder="1234-5678-9000-1111"
+                maxLength="16"
+                onChange={checksTypeNumber}
+              ></input>
+            </span>
+            <span>
+              <label htmlFor="expdate">Expiry date</label>
+              <input
+                id="expdate"
+                type="text"
+                placeholder="12/20"
+                onChange={checksTypeNumber}
+                maxLength="5"
+              ></input>
+            </span>
+            <span>
+              <label for="cvv">CVV</label>
+              <input
+                id="cvv"
+                type="text"
+                placeholder="456"
+                onChange={checksTypeNumber}
+                maxLength="3"
+              ></input>
+            </span>
+            {/* <PaymentInputs /> */}
+          </div>
+        </div>
+
+        <p className="total">
+          Total price of items in cart: {cookie.getJSON('total')}€
```

Comment:
Same feedback about the cookie code here - maybe makes sense to move it to a separate `cookie.ts` file.

Reply:
Oh and here, you can actually get the cookie earlier in `getServerSideProps` - since you're
already getting all of the cookies there.